README.md

```diff
@@ -47,19 +47,19 @@ It includes a mix of EDICS topics:
 - [HLT-UNDE] Spoken Language Understanding and Computational Semantics
 - [HLT-MLMD] Machine Learning Methods for Language
 
-1. S. F. Chen and J. Goodman, “An empirical study of smoothing techniques for language modeling”, Computer Speech & Language, 1999. (_N-Gram_)
+1. **An empirical study of smoothing techniques for language modeling**. Stanley F. Chen, Joshua Goodman. Computer Speech & Language, 1999. [PDF](https://pdf.sciencedirectassets.com/272453/1-s2.0-S0885230800X00101/1-s2.0-S0885230899901286/main.pdf?X-Amz-Security-Token=IQoJb3JpZ2luX2VjEOb%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJIMEYCIQDVcIa0J0HtvyOqvaJBRBwlAMjhEBNzsX%2BwubddBMfZCQIhAPowZZW%2F9jsb5lZiJO23mFyNkbRoJjQ%2Bx%2ByxkgabHeYwKrsFCM%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEQBRoMMDU5MDAzNTQ2ODY1IgwM8ztZNoMRY0mnLycqjwUkbMprjuXoyVZEEIcygqRc9%2BgmofK%2FyRxzaqlUi5EzU09dQCF3YJ7Awhy93HOiygCmJYkN3awOaYdgYSZGzA5vS2TQwIOjNeErQGgsBwUlPM%2F28gw1PgM%2B8ncO6hfVHWZqlLrQqy9TnXCpBC9CQBiDl2L98ARVApZAjDdCy2FD1dASM3K3NPCsx7c4I3NVy%2Bp0vL%2FL2WiPmFE6osBFTRHuQUVIkrGs05G3536NilhohvIijKOUcV6dZZrZX6kxjygVaYja2XbTN5BF2Xy7fa5pThjkfRMvd0xHiyoBgO%2FpK%2F7ShtiDUGqQnpYLCxkV9h4SV3P2%2F4NETYRwJ7EeKZWEpPp5CaEhk0ZNxoq%2BpBfXNeSgophxNJzS8jrgFMKhDhC%2B7%2BiDASC4Dv4zmr4PgiM%2BVOVkLwT8X7nfbEhmlaETbQAN3MMK5A9Kqk8OFHCsGEyIMvu1b1gl0bvTulNXPJ0jfApWKx2HzzkdtypDbRCgtIF%2BaIpbvvBqHPt3y%2FFfwoUBaYENRTqy1w62vJwbYnWgcfdVKraaK9d%2FBprp3RuxxrOIin0UvAUu67E1gj5F3rfWRf4kI1h8URqEZBvfnA3KfrdYbjxSqH28nMqSUtaXruVKRonmN10hVCBfrWN9roftbUSbvjguqIf94E36i4EwREvebljGPkpDH5J1R8oGd7HNf8QGb5Ewt6IlZynCJwDBceVevPKxc%2FtpcCk7XhVxjVR8gHfZTSyF2R2gj%2BopvUwtgGUH2Sv%2BMBoIvWSsBUNPxPEAaHS37FtZo7NNBhLAt3kADaxMek8GaOejzkquiG153rd8gvrApeGJ2%2FvZFhX5sz%2BN2XC48akiMoVqDgaYeXNxgGiRpn5vX15x%2Fj8hMIDo9a4GOrABv1Su0FE2RGfOzeLtPl%2F2yCH2pvBIZkRwY7Iwjh48fHLYZja1D1cX41DQbGo8dngQnMU3cPzYg%2BIgxRjUTLdK%2BjUtI4pRT06bIqMGYC3Ci3KAZVXqHGIJGbo3gQlBQxWY5U3XNf6ieMdkPa9T7p9kcnNQEyatKpTw5bOByQEDM%2BUVS2r45tnwt7zzbUAMGLb%2F9DhVBp95764bKOkCzMcobYpTRMDlylgnKTcOgwkQAoA%3D&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20240227T072541Z&X-Amz-SignedHeaders=host&X-Amz-Expires=300&X-Amz-Credential=ASIAQ3PHCVTYSYAKXHBH%2F20240227%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=c15c40de46d07af89aa02c48bba3e4b45746025b0d077953d15fe3a0a5620960&hash=565d4ee56b0aa58784d1203132e63688abec68d64ee3d594240eaa8f43f3079a&host=68042c943591013ac2b2430a89b270f6af2c76d8dfd086a07176afe7c76c2c61&pii=S0885230899901286&tid=spdf-45d54c50-a162-4ab2-a7e0-6f4c3effc9ff&sid=54e4f1ad7f19804c33895b44ed573490bd9fgxrqa&type=client&tsoh=d3d3LnNjaWVuY2VkaXJlY3QuY29t&ua=10125a510d0501520001&rr=85beb1ffb8a66871&cc=jp) (_N-Gram_)

```

Comment:
http://oa.ee.tsinghua.edu.cn/~ouzhijian/pgm/pgm-pdf/Chen&Goodman1998_An empirical study of smoothing techniques for language modeling.pdf